content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -118,7 +118,8 @@ different Kubernetes components.
 | `ExpandPersistentVolumes` | `true` | Beta | 1.11 | |
 | `ExperimentalHostUserNamespaceDefaulting` | `false` | Beta | 1.5 | |
 | `GenericEphemeralVolume` | `false` | Alpha | 1.19 | |
-| `GracefulNodeShutdown` | `false` | Alpha | 1.20 | |
+| `GracefulNodeShutdown` | `false` | Alpha | 1.20 | 1.20 |
+| `GracefulNodeShutdown` | `true` | Beta | 1.21 | |
```

Comment:
I'm afraid these these need to move to the section [Feature gates for graduated or deprecated features](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-graduated-or-deprecated-features)